packages/edit-site/src/components/sidebar-navigation-screen/index.js

```diff
@@ -90,7 +90,7 @@ export default function SidebarNavigationScreen( {
 					) }
 					<Heading
 						className="edit-site-sidebar-navigation-screen__title"
-						color={ 'white' }
+						color={ '#e0e0e0' /* $gray-200 */ }
```

Comment:
This isn't really ideal, but it's the best we can do for now without using an `!important` rule in the css.